openquake/hmtk/seismicity/declusterer/dec_gardner_knopoff.py

```diff
@@ -155,3 +158,30 @@ def decluster(self, catalogue, config):
         flagvector = flagvector[id1]
 
         return vcl, flagvector
+
+    @staticmethod
+    def get_distance(longitude, latitude, lon_i, lat_i, depth=None,
+                     depth_i=None):
+        """
+        Uses epicentral distance
+        """
+        return haversine(longitude, latitude, lon_i, lat_i).flatten()
+
+
+@DECLUSTERER_METHODS.add(
+    "decluster",
+    time_distance_window=TIME_DISTANCE_WINDOW_FUNCTIONS,
+    fs_time_prop=np.float)
+class GardnerKnopoffType1_3D(GardnerKnopoffType1):
+    """
+    Adaptation of the Gardner & Knopoff Type 1 algorithim taking into account
+    depth
+    """
+    @staticmethod
+    def get_distance(longitude, latitude, lon_i, lat_i, depth=None,
```

Comment:
This function is repeated. Wouldn't it be more efficient to put it in a utils.py file?